oauth2_provider/forms.py

```diff
@@ -80,7 +80,7 @@ def clean(self):
             except User.DoesNotExist:
                 user = None
 
-        if user is None or not user.is_active:
```

Comment:
@clintonb I agree about the `is_active` flag.  The DedX team has a story for this quarter to decouple the `is_active` and `email_verified` flags as part of work to remove gating of email validation on subsequent login.